docs/prob/api/provider/invalid-provider-name.md

```diff
@@ -5,7 +5,7 @@
 <!-- div:right-panel -->
 
 - Title: `Invalid provider name.`
-- Type: `/prob/api/provider/invalid-name`
+- Type: `/prob/api/invalid-provider-name`
```

Comment:
Type and file name doesn't match. Suggest renaming the type **and filename** back to

```suggestion
- Type: `/prob/api/provider/invalid-name`
```

as it falls into validation specifically for the provider group, and as we have a `/prob/api/project` namespace for validation errors regarding the project group.